javascript/javascript3/week3/Shopping cart/prod.js

```diff
@@ -0,0 +1,139 @@
+console.log("AB9");
+//Shopping cart using Classes
+
+// -- html elements
+const prodName = document.getElementById("prod-name");
+const prodPrice = document.getElementById("prod-price");
+const addBtn = document.querySelector(".add-prod");
+const searchBtn = document.querySelector(".search-product-btn");
+const searchProduct = document.getElementById("search-product-input");
+const body = document.querySelector("body");
+const productsSection = document.querySelector(".products");
+const appendShowDiv = document.querySelector(".show-product");
+const totalh4 = document.createElement("h4");
+
+///
+body.append(productsSection);
+///
+
+// CLASSES
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+}
+
+class ShoppingCart {
+  constructor() {
+    this.products = [];
+    this.username;
+  }
+
+  addProduct(product) {
+    this.products.push(product);
+  }
+
+  removeProduct(product) {
+    this.products.pop(product);
```

Comment:
it always removes the last item on the list, what if you want to remove a specific product with the name. you can use splice. read more about it here
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)